internal/librarian/release_init.go

```diff
@@ -201,23 +201,8 @@ func updateLibrary(repo gitrepo.Repository, library *config.LibraryState, librar
 		return fmt.Errorf("failed to fetch conventional commits for library, %s: %w", library.ID, err)
 	}
 
-	changes := make([]*config.Change, 0)
-	for _, commit := range commits {
-		clNum := ""
-		if cl, ok := commit.Footers[KeyClNum]; ok {
-			clNum = cl
-		}
-
-		changeType := getChangeType(commit)
-		changes = append(changes, &config.Change{
-			Type:       changeType,
-			Subject:    commit.Description,
-			Body:       commit.Body,
-			ClNum:      clNum,
-			CommitHash: commit.SHA,
-		})
-	}
-	if len(changes) == 0 {
+	updateLibraryChanges(library, commits)
+	if len(library.Changes) == 0 {
 		// Don't update the library at all
```

Comment:
Can you add a log message? As seen in Release Please, we will have lots of question "Why this doesn't make a release pull request?"